src/bacnet/basic/object/netport.h

```diff
@@ -207,16 +207,11 @@ BACNET_STACK_EXPORT
 bool Network_Port_BBMD_FD_Table_Set(uint32_t object_instance, void *fdt_head);
 
 BACNET_STACK_EXPORT
-bool Network_Port_Remote_BBMD_IP_Address(
```

Comment:
The IP address API is still valid for implementations that do not support DNS lookup, so we should not remove these in the header file.

Reply:
There are two things into it: #1. It is fetchable through Network_Port_Remote_BBMD_Address that can be Null(0), IP(1) or HostName[2] #2. Background: “e.g” 135.1 – 7.3.2.46.1.1 Clause J.1.2.1 requires that “B/IP devices shall be capable of resolving internet host names to IP addresses”. I can share the full exchange with the BTL labs....

Reply:
restored.